README.md

```diff
@@ -42,6 +43,6 @@ Ideally, you need two hosts to run this project:
 
 Now you can proceed to any of the following tasks:
 
- * [Cloning a Satellite server](docs/satellite-clone.md)
+ * [Cloning a Satellite server (or) Migrating a Satellite server from rhel6 to rhel7](docs/satellite-clone.md)
```

Comment:
My first question after seeing this would be "Does a migration still clone a Satellite"?